dribdat/templates/public/project.html

```diff
@@ -202,7 +207,7 @@ <h2>{{s.title}}</h2>
           </div>
           {% endif %}
           {% if s.text %}
-            {{s.text|markdown}}
+            {{s.text|oembed(html=False, maxwidth=600, maxheight=400)|markdown|onebox|safe}}
```

Comment:
Performance impact could be significant for a long list of log entries, and I don't think users will really put embeds or oneboxes in their posts.